src/screens/ProfileScreen/index.js

```diff
@@ -708,6 +708,9 @@ const ProfileScreen = ({route}) => {
               source={SOURCE_MY_PROFILE}
               hideThreeDot={false}
               showAnonymousOption={true}
+              onDeletePost={() => onDeletePost()}
+              isSelf={item.is_self}
+              isShowDelete={true}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `onDeletePost` and `isShowDelete` props have been added to the component. Ensure that these props are properly handled in the component definition.
```diff
+ onDeletePost={() => onDeletePost()}
+ isShowDelete={true}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->